src/cms/views/pages/page_tree_view.py

```diff
@@ -93,15 +88,60 @@ def get(self, request, *args, **kwargs):
                 request, _("You don't have the permission to edit or create pages.")
             )
         context = self.get_context_data(**kwargs)
+
+        pages = region.get_pages(archived=self.archived)
+        enable_drag_and_drop = True
+        # Filter pages according to given filters, if any
+        filter_data = kwargs.get("filter_data")
+        if filter_data is not None:
+            # Set data for filter form rendering
+            filter_form = PageFilterForm(data=filter_data)
+            if filter_form.is_valid():
+                # only filter if at least one checkbox but not all are checked
+                if (
+                    0
+                    < len(filter_form.cleaned_data["translation_status"])
+                    < len(translation_status.CHOICES)
+                ):
+                    enable_drag_and_drop = False
+                    utd = (
```

Comment:
```suggestion
                    filter_up_to_date = (
```
I prefer speaking variable names. Even if it's not very ambiguous in this case, there is no reason to be stingy with letters :wink: